File: src/Api/Model/DB/TipoSocio.cs
using Newtonsoft.Json;
using Realms;
using System;
using System.Collections.Generic;
using System.Text;

namespace CriadorBaseDados.Model.DB
{
    [JsonObject(MemberSerialization.OptIn)]
    public class TipoSocio : RealmObject
    {
        /// <summary>
        /// 1 – PESSOA JURÍDICA
        //  2 – PESSOA FISICA
        //  3 – ESTRANGEIRO
        /// </summary>
        [JsonProperty]
        [PrimaryKey]
        public int Cod { get; set; }
        [JsonProperty]
        public string Descricao { get; set; }
    }
}

File: src/Mobile/b2b/lib/model/core/tipo_socio.dart
class TipoSocio {
  /// <summary>
  /// 1 – PESSOA JURÍDICA
  //  2 – PESSOA FISICA
  //  3 – ESTRANGEIRO
  /// </summary>
  final int cod;
  final String descricao;

  TipoSocio(this.cod, this.descricao);

  TipoSocio.fromJson(Map<String, dynamic> json)
      : cod = json['cod'],
        descricao = json['descricao'];

  Map<String, dynamic> toMap() => <String, dynamic>{
        'cod': cod,
        'descricao': descricao,
      };
}
